modules/apps/frontend-data-set/frontend-data-set-views-web/npmscripts.config.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+module.exports = {
+	build: {
+		main: 'src/main/resources/META-INF/resources/js/index.ts',
```

Comment:
I've had to migrate `frontend-data-set-views-web` to ESM to make the dynamic `import` work, since the AMD build removes it :-/

Reply:
this shouldn't be an issue as nobody should import stuff we have in this module, except us in this very same module.

Reply:
yes